pandas/core/dtypes/cast.py

```diff
@@ -159,6 +159,31 @@ def maybe_box_datetimelike(value: Scalar, dtype: Optional[Dtype] = None) -> Scal
     return value
 
 
+# TODO: this should be a TypeVar
+def maybe_box_native(value: Union[ABCSeries, Scalar]) -> Union[ABCSeries, Scalar]:
```

Comment:
why would this take a Series?

Reply:
When calling `to_dict` with `orient=series`:
``` python
In [2]: df = pd.DataFrame({"a": range(3), "b": range(3,6)})

In [3]: df.to_dict(orient="series")
Out[3]: 
{'a': 0    0
 1    1
 2    2
 Name: a, dtype: int64,
 'b': 0    3
 1    4
 2    5
 Name: b, dtype: int64}
```

Reply:
is `ABCSeries` a valid type? seems mypy is throwing an error

Reply:
Thanks for pointing this out! It's not - I've switched to using `Series` instead